library/src/main/java/jp/co/cyberagent/android/gpuimage/GPUImage.java

```diff
@@ -143,9 +174,12 @@ public void setUpCamera(final Camera camera) {
      */
     public void setUpCamera(final Camera camera, final int degrees, final boolean flipHorizontal,
```

Comment:
This method can be `package-private` because `setUpCamera` must be called via `GPUImageView`.